integrations/watsonx/tests/test_chat_generator.py

```diff
@@ -0,0 +1,305 @@
+import logging
```

Comment:
Let's update this to 
```suggestion
from haystack import logging
```